### ISSUE TITLE
Problems when trying to upgrade the project to Windows 10 Universal App.

### ISSUE THREAD

**Issue by pradeepgaba** [phatware/WritePadSDK#3]
Hi , I was creating application for windows 8.1 that was using writing pad sdk and I was using the solution that was given there in the SDK source code. But the problem came when i tried to use the SDK on windows 10 higher version.  I added a project that targeted the windows universal app. I have added all the needed references that needed to run the project but every time i run the windows 10 universal project i get error Unable to load DLL 'WinRT_CPPLayer.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E). 

Please help.
![error](https://cloud.githubusercontent.com/assets/6142282/18044150/f7cc9b38-6de9-11e6-97b2-1ad1c3fcd843.JPG)

**Comment by phatware** [comment]
Most likely the WinRT_CPPLayer.dll was either not compiled (it is compiled separably from C# project) or not in the search path.

**Comment by pradeepgaba** [comment]
I found the solution. It was run time  that was missing. I added it and it worked

**Comment by pradeepgaba** [comment]
But i found another problem when i run the app in debug mode it works without any error but when i create app package and deploy it it crashes right after i try to load the api for writing pad. I tried to create app package and deploy it without calling this api it works but not in case i am calling api.

I inspected the Windows Event log to check for errors also which is given bellow:-

Faulting application name: MyApp.Windows10.exe, version: 1.0.0.0, time stamp: 0x56c69c5d
Faulting module name: edgehtml.dll, version: 11.0.10240.17113, time stamp: 0x57cf9993
Exception code: 0xc0000005
Fault offset: 0x000000000008f179
Faulting process id: 0x3084
Faulting application start time: 0x01d20ffe121d5a88
Faulting application path: C:\Program Files\WindowsApps\43127Gabas.5358389801D2D_1.1.2.0_x64__dt3a6vmkpr4b2\MyApp.Windows10.exe
Faulting module path: C:\Windows\SYSTEM32\edgehtml.dll
Report Id: ba36073b-84fe-443d-9dde-59ca7d62aac4
Faulting package full name: 43127Gabas.5358389801D2D_1.1.2.0_x64__dt3a6vmkpr4b2
Faulting package-relative application ID: App

Note I am using this api in windows 10 universal apps. It is working fine when using it in windows 8.1  but not in windows 10 universal app when deploying using package. App is crashing.
